build/Dockerfile.rpm

```diff
@@ -6,21 +6,24 @@
 
 FROM      centos:6.8
 
-RUN yum -y update
-RUN yum groupinstall -y development
-RUN yum -y install fakeroot mysql mysql-devel postgresql-devel lm_sensors-devel cmake screen which tar wget
-RUN rpm -e git perl-Git
-RUN yum -y install curl-devel expat-devel gettext-devel openssl-devel zlib-devel
-RUN yum -y install gcc perl-ExtUtils-MakeMaker
-RUN wget https://www.kernel.org/pub/software/scm/git/git-2.9.2.tar.gz
-RUN tar xzf git-2.9.2.tar.gz
-RUN cd git-2.9.2; make prefix=/usr; make prefix= /usr install
-RUN git config --global user.email "you@example.com"
-RUN git config --global user.name "Your Name"
+RUN yum -y update \
+ && yum groupinstall -y development \
+ && yum -y install fakeroot mysql mysql-devel postgresql-devel lm_sensors-devel cmake screen which tar wget \
+ && rpm -e git perl-Git \
+ && yum -y install curl-devel expat-devel gettext-devel openssl-devel zlib-devel \
+ && yum -y install gcc perl-ExtUtils-MakeMaker \
+ && wget https://www.kernel.org/pub/software/scm/git/git-2.9.2.tar.gz \
+ && tar xzf git-2.9.2.tar.gz \
+ && rm -f git-2.9.2.tar.gz \
+ && yum clean all
```

Comment:
do we want to add 'apt-get clean' for debian:7 since we added 'yum clean all' here.

Reply:
good point, i also saw a couple more that could use some cleanup. adding them now...